server-spi-private/src/main/java/org/keycloak/migration/MigrationModelManager.java

```diff
@@ -17,169 +17,24 @@
 
 package org.keycloak.migration;
 
-import java.util.LinkedHashMap;
-import java.util.Map;
-import java.util.regex.Pattern;
-import org.jboss.logging.Logger;
-import org.keycloak.common.Version;
-import org.keycloak.migration.migrators.MigrateTo12_0_0;
-import org.keycloak.migration.migrators.MigrateTo14_0_0;
-import org.keycloak.migration.migrators.MigrateTo18_0_0;
-import org.keycloak.migration.migrators.MigrateTo1_2_0;
-import org.keycloak.migration.migrators.MigrateTo1_3_0;
-import org.keycloak.migration.migrators.MigrateTo1_4_0;
-import org.keycloak.migration.migrators.MigrateTo1_5_0;
-import org.keycloak.migration.migrators.MigrateTo1_6_0;
-import org.keycloak.migration.migrators.MigrateTo1_7_0;
-import org.keycloak.migration.migrators.MigrateTo1_8_0;
-import org.keycloak.migration.migrators.MigrateTo1_9_0;
-import org.keycloak.migration.migrators.MigrateTo1_9_2;
-import org.keycloak.migration.migrators.MigrateTo2_0_0;
-import org.keycloak.migration.migrators.MigrateTo2_1_0;
-import org.keycloak.migration.migrators.MigrateTo2_2_0;
-import org.keycloak.migration.migrators.MigrateTo2_3_0;
-import org.keycloak.migration.migrators.MigrateTo2_5_0;
-import org.keycloak.migration.migrators.MigrateTo3_0_0;
-import org.keycloak.migration.migrators.MigrateTo3_1_0;
-import org.keycloak.migration.migrators.MigrateTo3_2_0;
-import org.keycloak.migration.migrators.MigrateTo3_4_0;
-import org.keycloak.migration.migrators.MigrateTo3_4_1;
-import org.keycloak.migration.migrators.MigrateTo3_4_2;
-import org.keycloak.migration.migrators.MigrateTo4_0_0;
-import org.keycloak.migration.migrators.MigrateTo4_2_0;
-import org.keycloak.migration.migrators.MigrateTo4_6_0;
-import org.keycloak.migration.migrators.MigrateTo6_0_0;
-import org.keycloak.migration.migrators.MigrateTo8_0_0;
-import org.keycloak.migration.migrators.MigrateTo8_0_2;
-import org.keycloak.migration.migrators.MigrateTo9_0_0;
-import org.keycloak.migration.migrators.MigrateTo9_0_4;
-import org.keycloak.migration.migrators.Migration;
-import org.keycloak.models.Constants;
-import org.keycloak.models.DeploymentStateProvider;
 import org.keycloak.models.KeycloakSession;
 import org.keycloak.models.RealmModel;
 import org.keycloak.representations.idm.RealmRepresentation;
+import org.keycloak.storage.DatastoreProvider;
 
 /**
  * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
  * @version $Revision: 1 $
  */
 public class MigrationModelManager {
```

Comment:
Please add deprecation note that this class will be moved

Reply:
added in next commit